src/main/java/com/gd/clinic/entities/Doctor.java

```diff
@@ -0,0 +1,29 @@
+package com.gd.clinic.entities;
+
+import com.sun.xml.bind.v2.TODO;
+import lombok.*;
+import org.hibernate.annotations.Type;
+
+import javax.persistence.*;
+import java.util.UUID;
+
+@NoArgsConstructor
+@RequiredArgsConstructor
```

Comment:
this you also don't need

Reply:
That's right, was thinking  objects from the class might be instantiated in the future. Since they're actually not being used, they've been removed this commit.